lib/model/message_list.dart

```diff
@@ -383,6 +383,7 @@ class MessageListView with ChangeNotifier, _MessageSequence {
       if (_messageVisible(message)) {
         _addMessage(message);
       }
+      store.recentSenders.handleMessage(message);
```

Comment:
```
autocomplete: Add "recent activity in current topic/stream" criterion
```

The changes in this file in this commit look like they belong in an earlier commit:

```
store: Add `RecentSenders` data structure
```

That earlier commit calls `recentSenders.handleMessage` in one place it needs to, but these are places where it needs to be called too.